tests/testdata/control_files/3d/expected_ambient_occlusion_1/expected_framegraph.txt

```diff
@@ -2,43 +2,44 @@
   (Qt3DRender::QViewport{17/<no_name>})
     (Qt3DRender::QCameraSelector{18/Forward render pass CameraSelector}) [ (Qt3DRender::QCamera:{0/<no_name>}) ]
       (Qt3DRender::QLayerFilter{19/<no_name>}) [ (AcceptAnyMatchingLayers:[ {12/<no_name>} ]) ]
-        (Qt3DRender::QRenderTargetSelector{25/<no_name>}) [ (outputs:[ (Depth:{21[DepthFormat]/<no_name>), (Color0:{20[RGB8_UNorm]/<no_name>) ]) ]
```

Comment:
I would like to suggest again that we remove node IDs from the framegraph dump, because it is close to impossible to see actual changes in the diff. The node IDs do not add any value, only noise...

Reply:
It help, for example, in understanding texture usage between scenegraph and framegraph or the order of object creation. In tests, ids are less important but it needs an other PR.

Reply:
Indeed, that's for another PR, it just caught my eye again :slightly_smiling_face: 
(by the way, the order of object creation hopefully(!) should not matter, and for understanding texture usage we could rather use object names)

Reply:
Will be covered in an other PR.